spec/commands/asset/delete_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'cache/asset'
+require 'filesystem'
+require 'commands'
+require 'utils'
+
+RSpec.describe Metalware::Commands::Asset::Delete do
+
+  it 'errors if the asset doesnt exist' do
```

Comment:
So the `doesnt` needs the apostrophe. Either convert the string quotes to double quotes, or better yet, expand the contraction to `does not`.